spaceship/spec/tunes/availability_spec.rb

```diff
@@ -68,6 +68,28 @@
         expect(availability.territories.length).to eq(1)
         expect(availability.territories[0].code).to eq("US")
       end
+
+      it "correctly sets preorder with no app available date" do
+        TunesStubbing.itc_stub_set_preorder_cleared
```

Comment:
Nice. We're using the older-style stubbing here, but it's close to the test site 👍